packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -129,6 +129,27 @@ const installNodeModulesTasks = ({ newAppDir }) => {
         })
       },
     },
+    {
+      title: 'Updating Prisma seed file imports',
```

Comment:
This task replaces the first 2 lines of the seed.ts with `require`s and renames the seed.ts to seed.js.

Perhaps instead can use babel?